Lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -2040,11 +2039,7 @@ class ByteCodeBufferBuilder
                 "Only the ErrorOnNew|SuperReference|Lambda|CapturesThis|Generator|ClassConstructor|Async|ClassMember attributes should be set on a serialized function");
 
         PrependInt32(builder, L"Offset Into Source", sourceDiff);
-        if (function->GetNestedCount() > 0)
-        {
-            definedFields.has_m_nestedCount = true;
-            PrependInt32(builder, L"Nested count", function->GetNestedCount());
-        }
+        PrependInt32(builder, L"Nested count", function->GetNestedCount());
```

Comment:
m_nestedCount is removed from the functionbody,  added a has_m_nestedCount in serializedFieldList and get the 4 bytes back